scripts/globals/crafting.lua

```diff
@@ -10,6 +10,7 @@ require('scripts/globals/utils')
 xi = xi or {}
 xi.crafting = {}
 
+-- This IDs are probably made-up.
```

Comment:
Probably not necessary to comment this if we're only using this internally

Reply:
Well, turns out SE does have real guild IDs. Since I am going to cleanup further this file to actually align with retail instead of making up event parameters and such, I put that comment in there for me, mostly so I don't forget.

Reply:
I missed that comment line altogether or I'd have let you know they weren't.